# Question 2E

In [1]:
dollars = c(11,18,17,15,9,5,12,19,22,25)
satisfaction = c(6,8,10,4,9,6,3,5,2,10)
reg = lm(satisfaction ~ dollars)
summary(reg)


Call:
lm(formula = satisfaction ~ dollars)

Residuals:
    Min      1Q  Median      3Q     Max 
-4.5398 -2.0751 -0.0387  2.5949  3.6392 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)  
(Intercept)  5.75244    2.70119   2.130   0.0658 .
dollars      0.03579    0.16503   0.217   0.8337  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 3.035 on 8 degrees of freedom
Multiple R-squared:  0.005844,	Adjusted R-squared:  -0.1184 
F-statistic: 0.04703 on 1 and 8 DF,  p-value: 0.8337


# Question 3A

In [2]:
age = c(18,20,22,23,23,25,27,28,29,32,37,41,46,49,53,55,
        63,65,66,67,68,70,71,72,73,74,75,77,79,82)
distance = c(510,590,560,510,460,490,560,510,460,410,
             420,460,450,380,460,420,350,420,300,410,
             300,390,320,370,280,420,460,360,310,360)
reg = lm(distance ~ age)
summary(reg)


Call:
lm(formula = distance ~ age)

Residuals:
    Min      1Q  Median      3Q     Max 
-78.231 -41.710   7.646  33.552 108.831 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept) 576.6819    23.4709  24.570  < 2e-16 ***
age          -3.0068     0.4243  -7.086 1.04e-07 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 49.76 on 28 degrees of freedom
Multiple R-squared:  0.642,	Adjusted R-squared:  0.6292 
F-statistic: 50.21 on 1 and 28 DF,  p-value: 1.041e-07


Least Squares Estimate 

Y = 576.6819 - 3.0068x

# Question 3B

The standard deviation of the error term is the residual standard error of 49.76

# Question 3C

H0: B1 = 0
Ha: B1 does not equal 0

alpha = 0.05

In [3]:
#find t(1-alpha/2, n-2)
qt(0.975,28)

[1] 2.048407

In [5]:
#as seen in the regression model, t* = -7.086
abs(-7.086)

[1] 7.086

We should reject the null hypothesis because t* = 7.086 > t = 2.048. Because we reject the null hypothesis, we can conclude that there is a significant correlation between age and distance, which means that the model is useful. 

# Question 3D

The R^2 value is the multiple r-squared value of 0.642. This value means that 64.2% of the total sample variance can be explained by the linear relationship between age and distance. This suggests that the model is useful because the value is high; however, the value also shows that 35.8% of the sample variance cannot be explained by the model. Overall, the R^2 value confirms that the model is useful.

# Question 3E

In [6]:
cor(age,distance)

[1] -0.8012447

The correlation coefficient is -0.8012. This value suggests that age and distance are variables that have a strong, negative correlation. Because the correlation coefficient is strong, it suggests that the model is useful.

# Question 3F

In [7]:
576.6819-3.0068*50

[1] 426.3419

I would predict the distance that a 50-year-old driver can see to be 426.3419.

# Question 3G

In [8]:
age = c(18,20,22,23,23,25,27,28,29,32,37,41,46,
        49,53,55,63,65,66,67,68,70,71,72,73,74,75,77,79,82)
distance = c(510,590,560,510,460,490,560,510,460,410,420,460,
             450,380,460,420,350,420,300,410,300,390,320,370,
             280,420,460,360,310,360)
reg = lm(distance ~ age)
predict(reg, data.frame(age = 50), interval = "p")

fit      lwr      upr     
1 426.3402 322.7196 529.9607

# Question 3H

In [9]:
age = c(18,20,22,23,23,25,27,28,29,32,37,41,46,
        49,53,55,63,65,66,67,68,70,71,72,73,74,75,77,79,82)
distance = c(510,590,560,510,460,490,560,510,460,410,420,460,
             450,380,460,420,350,420,300,410,300,390,320,370,
             280,420,460,360,310,360)
reg = lm(distance ~ age)
predict(reg, data.frame(age = 50), interval = "c")

fit      lwr      upr     
1 426.3402 407.7097 444.9706